### ISSUE TITLE
MSBuildGitHashSuppressDirtyWarning not working in 2.0.0

### ISSUE THREAD

**Issue by dotnetshadow** [MarkPflug/MSBuildGitHash#22]
Hi there,

A great library, I noticed in v2.0.0 that ```<MSBuildGitHashSuppressDirtyWarning>true</MSBuildGitHashSuppressDirtyWarning>``` no longer works and the message gets emitted

As a sidenote I noticed when using Resharper I tend to get messages on the bottom stating **sync solution** I think it's every time a file might be getting changed while developing. Not sure if it's related, but if it is, is there is a way to prevent MSBuildGitHash from running locally?

**Comment by MarkPflug** [comment]
Thanks for bringing this to my attention. That `MSBuildGitHashSuppressDirtyWarning` got lost in the 2.0.0 refactor. I just pushed 2.0.1 to nuget which restores this feature.

As far as the second issue, there is no in-built way to turn off MSBuildGitHash, but you can probably do it within your own projects though.
Somewhere in your project you'll have this line: (I'm assuming you are using packageReference and not packages.config)
```
<ItemGroup>
<PacakageReference Include="MSBuildGitHash" Version="2.0.1"/>
</ItemGroup>
```

You can add this line later in the file (or in a [Directory.Build.targets](https://docs.microsoft.com/en-us/visualstudio/msbuild/customize-your-build?view=vs-2019)):
```
<ItemGroup Condition="'$(BuildingInsideVisualStudio)' == 'true'">
<PacakageReference Remove="MSBuildGitHash"/>
</ItemGroup>
```

I don't use resharper, so I have no way of testing if this will fix the issue you are seeing. If it does, please let me know so I can add it to a readme for other people who might see the same thing.

**Comment by dotnetshadow** [comment]
> Thanks for bringing this to my attention. That `MSBuildGitHashSuppressDirtyWarning` got lost in the 2.0.0 refactor. I just pushed 2.0.1 to nuget which restores this feature.

This is now working thank you very much.


**Another Issue** 
I noticed that the information is not being injected in the file properties, doesn't matter what I try.
I'm building from dotnet cli ```dotnet build```

**EDIT I think it was because I was using a preview version of ```dotnet``` which got installed because of a the latest visual studio preview. When working with the released version everything works** 

I can see it being outputed in the console:
![image](https://user-images.githubusercontent.com/296247/79035724-9503f780-7c04-11ea-94ff-67a3d4d8f01f.png)

But the information is missing in the file properties
![image](https://user-images.githubusercontent.com/296247/79035740-b1a02f80-7c04-11ea-9b8d-8c46ddde1e08.png)

I guess I'm missing how this is supposed to work? Basically I committed, pushed to remote branch. Then I made another update locally and I build the project. Should the file properties change? or only if I publish? Or only in release production mode?

> You can add this line later in the file (or in a [Directory.Build.targets](https://docs.microsoft.com/en-us/visualstudio/msbuild/customize-your-build?view=vs-2019)):
> 
> ```
> <ItemGroup Condition="'$(BuildingInsideVisualStudio)' == 'true'">
> <PacakageReference Remove="MSBuildGitHash"/>
> </ItemGroup>
> ```
Thanks for this recommendation, I'll see if it helps and report back.
